scapy/layers/bluetooth.py

```diff
@@ -1038,9 +1038,52 @@ class HCI_Cmd_Write_LE_Host_Support(Packet):
 
 
 # 7.4 INFORMATIONAL PARAMETERS, the OGF is defined as 0x04
+class HCI_Cmd_Read_Local_Version_Information(Packet):
+    """
+
+    7.4.1 Read Local Version Information command
+
+    """
+    name = "HCI_Read_Local_Version_Information"
```

Comment:
This packets don't have a field_desc. Why are they useful for you?

Reply:
Because they set an `opcode` or `ogf` and `ocf` in the parent packet.

They are part of the Bluetooth specification and have a function.